feat: create privacy policy

wldeh opened the pull request:
<!---
Thank you for your pull request! 🚀

If you're not already a member of our public Matrix community, please consider joining via the following link:
https://matrix.to/#/#activist_community:matrix.org

It'd be great to have you!

Also if you're new to open source, check that the email you use for GitHub (https://github.com/settings/emails) is the same the one you have for git so you get credit for your contribution. You can see your git email by typing `git config user.email` and set it globally with `git config --global user.email "your_email@email.com"`.
-->

### Contributor checklist

<!-- Please replace the empty checkboxes [ ] below with checked ones [x] accordingly. -->

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
  <!-- - [ ] I have updated the [CHANGELOG](https://github.com/activist-org/activist/blob/main/CHANGELOG.md) with a description of my changes for the upcoming release (if necessary)  -->
  <!-- ... or I'll send a commit with this now 🙃 -->

---

### Description

<!--
Describe briefly what your pull request proposes to change. Especially if you have more than one commit, it is helpful to give a summary of what your contribution is trying to solve.

Also, please describe shortly how you tested that your change actually works.
-->
This PR adds a detailed privacy policy at /legal/privacy-policy UR

### Related issue

<!--- activist prefers that pull requests be related to already open issues. -->
<!--- If applicable, please link to the issue by replacing ISSUE_NUMBER with the appropriate number below. -->
<!--- Feel free to delete this section if this does not apply. -->

- #44

andrewtavis commented:
Thanks for your work here as well, @wldeh! I’ll check all this when you’ve finalized them :)

wldeh commented:
> Thanks for your work here as well, @wldeh! I’ll check all this when you’ve finalized them :)

For sure! Yeah I'm not sure if there were any plans to set up a Figma page for this project. Regarding the policy contents I've committed here, there's definitely room for improvement. I would appreciate any guidance or direction you may have. Looking forward to your feedback!

andrewtavis commented:
Thanks, @wldeh! There were no Figma files for this page specifically, but I'll likely style it the same as the about page once the text contents are merged. We can also do an actual `PRIVACY.txt` file for this as well like in Scribe :) I'll check this later!

wldeh commented:
> Thanks, @wldeh! There were no Figma files for this page specifically, but I'll likely style it the same as the about page once the text contents are merged. We can also do an actual `PRIVACY.txt` file for this as well like in Scribe :) I'll check this later!

Appreciate the update @andrewtavis! Are we planning to put the PRIVACY.txt file in the root directory? I can integrate it here soon after taking a closer look at the Scribe implementation.

andrewtavis commented:
Hey @wldeh 👋 Yes, the `PRIVACY.txt` file would go in the project root. If you have interest in integrating it, then please do 😊 I’m very focused on the v2.2.0 release of Scribe at the moment, but my priority will switch hard back to activist next week :) Really grateful to get some help here!

wldeh commented:
> Hey @wldeh 👋 Yes, the `PRIVACY.txt` file would go in the project root. If you have interest in integrating it, then please do 😊 I’m very focused on the v2.2.0 release of Scribe at the moment, but my priority will switch hard back to activist next week :) Really grateful to get some help here!

Completely understood. Alright yeah I'll definitely look to have a polished setup here by next week. Thanks again!

andrewtavis commented:
Thank you!! 😊

andrewtavis commented on frontend/pages/legal/privacy-policy.vue:
```diff
@@ -1,14 +1,147 @@
+            have regarding your information.
+          </p>
+          <p>
+            By using Activist.org, you agree to the provisions of this Privacy
```
activist is always lower case 🙃

andrewtavis commented on frontend/pages/legal/privacy-policy.vue:
```diff
@@ -1,14 +1,147 @@
+          <p>
+            We do not share your personal information with any third parties,
+            nor do we use any third-party services that collect or process
+            personal information. Activist.org is dedicated to ensuring the
```
Same here 🙃

andrewtavis commented on frontend/pages/legal/privacy-policy.vue:
```diff
@@ -1,14 +1,147 @@
+          </p>
+          <p>
+            We are committed to transparency and user control. We encourage
+            users to contribute to Activist.org and participate in discussions
```
And one below :)

andrewtavis commented on frontend/pages/legal/privacy-policy.vue:
```diff
@@ -1,14 +1,147 @@
+          <p>
+            We may update this Privacy Policy from time to time. If we make
+            significant changes, we will notify you through the platform or by
+            other means. By continuing to use Activist.org after any changes are
```
Last one!!! 😊

andrewtavis reviewed (changes requested):
This reads really well, @wldeh! I'll go through and do some edits on my side, but from my end we can just change the `Activist.org` mentions to `activist.org`. I can then go through and format the page like https://activist.org/en/about/. 

You're also welcome to commit an updated version as a `PRIVACY.txt` file! 

Thanks again, @wldeh! 🚀 Super helpful 😊

wldeh commented:
> Alsooo, if you're a Matrix user, feel free to join us in the [public chat](https://matrix.to/#/#activist_community:matrix.org) so we can connect on future issues 😊

Thank you for the link! I'm actually not a Matrix user yet, but I'm interested in learning more about it. Going to setup an account here shortly.

andrewtavis commented:
Great, @wldeh! I'm almosttt done with the Scribe release, btw 😊 So much to do for this one as the data process changed dramatically. Will get to this soon after :)

Feel free to send along the changes that I mentioned [here](https://github.com/activist-org/activist/pull/95#pullrequestreview-1384842984) and we can merge directly :) :)

wldeh commented:
> This reads really well, @wldeh! I'll go through and do some edits on my side, but from my end we can just change the `Activist.org` mentions to `activist.org`. I can then go through and format the page like https://activist.org/en/about/.
> 
> You're also welcome to commit an updated version as a `PRIVACY.txt` file! We can then merge after that :)
> 
> Thanks again, @wldeh! 🚀 Super helpful 😊



> This reads really well, @wldeh! I'll go through and do some edits on my side, but from my end we can just change the `Activist.org` mentions to `activist.org`. I can then go through and format the page like https://activist.org/en/about/.
> 
> You're also welcome to commit an updated version as a `PRIVACY.txt` file! We can then merge after that :)
> 
> Thanks again, @wldeh! 🚀 Super helpful 😊

Thank you for your feedback, @andrewtavis! I fixed all the Activist.org issues here and created a PRIVACY.txt file in the root directory. Ill try to push a commit here in the next day that replicates the 

> Great, @wldeh! I'm almosttt done with the Scribe release, btw 😊 So much to do for this one as the data process changed dramatically. Will get to this soon after :)
> 
> Feel free to send along the changes that I mentioned [here](https://github.com/activist-org/activist/pull/95#pullrequestreview-1384842984) and we can merge directly :) :)

Alright yeah I fixed all the Activist.org issues here and created a PRIVACY.txt file in the root directory. Ill try to push a commit here in the next day that replicates the about page as well. Thanks again for your feedback, @andrewtavis and apologies for the delay!

wldeh commented:
> No delay whatsoever, @wldeh! The stuff about formatting the page and everything is definitely beyond what I'd expect 😊👏 Thank you! What you can do is just use the about pages icon and everything, so just copy that page and replace the content with the policy. I'll then go in and replace the activist icon with something like a shield or something like we have for Scribe :)

Perfect Alright. Just pushed these changes. Things are not looking the cleanest but feel free to review when you can.

andrewtavis reviewed (approved):
Thank you, @wldeh! Sorry for getting to this a bit late :) Has been a crazy busy week for me, but I'm shifting focus back to activist going forward 😊 I'll do some minor edits to the text and switch over the privacy policy icon to something to a shield with a checkmark so we're telling people they're safe.

Looking forward to further collaborations!

wldeh commented:
> Thank you, @wldeh! Sorry for getting to this a bit late :) Has been a crazy busy week for me, but I'm shifting focus back to activist going forward 😊 I'll do some minor edits to the text and switch over the privacy policy icon to something to a shield with a checkmark so we're telling people they're safe.
> 
> Looking forward to further collaborations!

Great to hear Andrew and no worries at all! The checkmark definitely makes more sense. Happy to say I am a fan of this projects vision and definitely planning on making some more significant contributions here moving forward. Cheers!

andrewtavis commented:
Wonderful to hear 😊😊
